Frontend/ui-common/src/Engineering/Helpers/Utils.purs

```diff
@@ -258,3 +261,23 @@ catMaybeStrings arr =
       case x of
         Just a -> acc <> a <> " "
         Nothing -> acc) "" arr
+
+capitalizeFirstChar :: String -> String
+capitalizeFirstChar inputStr =
+  let splitedArray = split (Pattern " ") (inputStr)
+      output = map (\item -> (toUpper (take 1 item)) <> (toLower (drop 1 item))) splitedArray
+    in joinWith " " output
+
+capitalizeOnlyFirstChar :: String -> String 
+capitalizeOnlyFirstChar inputStr =
+  let splitedArray = split (Pattern " ") (inputStr)
+      output = map (\item -> (toUpper (take 1 item)) <>  (drop 1 item)) splitedArray
+    in joinWith " " output
```

Comment:
use `capitalize` function from `MobilityPrelude` but what's the difference between these two functions

Reply:
it will capitalize the first char only and not convert tail part to lower case. it will be used for when there is abbreviations (ex: SOS)

Reply:
this is reduce code redundancy and improve readability. What's the use of these two functions can you provide sample input and outputs?